package.json

```diff
@@ -4,9 +4,9 @@
   "private": true,
   "scripts": {
     "start": "node ./bin/www",
-    "watch": "nodemon --watch ./ --watch views --watch controllers --watch build --watch helpers --ext js,hbs,json ./bin/www",
-    "debug": "nodemon --inspect=9310 --watch controllers --watch helpers --watch views --watch helpers --ext js,hbs,json ./bin/www",
-    "debug:container": "nodemon --inspect=0.0.0.0:9310 --watch controllers --watch helpers --watch views --watch helpers --ext js,hbs,json ./bin/www",
+    "watch": "nodemon --watch ./ --watch views --watch theme --watch controllers --watch build --watch helpers --ext js,hbs,json ./bin/www",
+    "debug": "nodemon --inspect=9310 --watch ./ --watch views --watch theme --watch controllers --watch build --watch helpers --ext js,hbs,json ./bin/www",
```

Comment:
either watch everything or nothing. however use nodemon file instead of defining all the directories twice. @adrianjost and do not watch build dir or other system directories

Reply:
@schurigh

Reply:
removed build

Reply:
build added again - needed for client to recognize new files from static folder.